src/connectors/toggle/connectToggle.js

```diff
@@ -275,6 +278,52 @@ export default function connectToggle(renderFn, unmountFn) {
 
         return nextState;
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`